src/libraries/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;netcoreapp3.0;net461</TargetFrameworks>
     <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
     <Nullable>enable</Nullable>
+    <ILLinkTrimAssembly>true</ILLinkTrimAssembly>
```

Comment:
What's the intention here? This runs the linker during the build to trim code. But it doesn't fail on warnings.

Reply:
N/A as discussed - good catch though, thanks.